specification/sqlvirtualmachine/resource-manager/readme.md

```diff
@@ -28,18 +28,27 @@ These are the global settings for the SQL virtual machine API.
 title: SqlVirtualMachineManagementClient
 description: 'The SQL virtual machine management API provides a RESTful set of web APIs that interact with Azure Compute, Network & Storage services to manage your SQL Server virtual machine. The API enables users to create, delete and retrieve a SQL virtual machine, SQL virtual machine group or availability group listener.'
 openapi-type: arm
-tag: package-preview-2021-11
+tag: package-2022-02
 ```
 
 
+### Tag: package-2022-02
+
+These settings apply only when `--tag=package-2022-02` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-02'
+input-file:
+  - Microsoft.SqlVirtualMachine/stable/2022-02-01/sqlvm.json
+```
 ### Tag: package-preview-2021-11
 
 These settings apply only when `--tag=package-preview-2021-11` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2021-11'
+``` yaml $(tag) == 'package-preview-2021-11'
```

Comment:
nit: this space is unecessary

Reply:
This is indented/generated by prettier-check. https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md#prettier-check